python/README.md

```diff
@@ -6,7 +6,7 @@ PyCarbon provides python API for integrating CarbonData with AI framework like
 
 $ git clone https://github.com/apache/carbondata.git
 
-$ cd python/pycarbon
```

Comment:
suggesion:  add pycarbon fold to line 17 and other place

Reply:
done